nautobot/extras/api/serializers.py

```diff
@@ -128,8 +133,14 @@ def get_field_names(self, declared_fields, info):
 
     @extend_schema_field(serializers.URLField())
     def get_notes_url(self, instance):
-        notes_url = get_route_for_model(instance, "notes", api=True)
-        return reverse(notes_url, args=[instance.id], request=self.context["request"])
+        try:
+            notes_url = get_route_for_model(instance, "notes", api=True)
+            return reverse(notes_url, args=[instance.id], request=self.context["request"])
+        except NoReverseMatch:
+            logger.warning(
+                "Notes feature is not available for this model. Please implement it before including NotesSerializerMixin"
+            )
+            return None
```

Comment:
I manually took the notes route out for `CircuitTermination`. This is what the terminal output and browsable API page look like. Im fairly confident it will solve the problem of `get_notes_url()` getting called on without the model having a notes route.
<img width="1033" alt="Screen Shot 2022-11-10 at 5 24 05 PM" src="https://user-images.githubusercontent.com/46973263/201218862-0455ae48-2c5d-4498-a171-fdb87ca51e96.png">
<img width="1279" alt="Screen Shot 2022-11-10 at 5 24 35 PM" src="https://user-images.githubusercontent.com/46973263/201218937-4cd0cc5c-4831-4023-a2ad-3500c41ab9c1.png">